src/modules/balance/ModuleBalanceCard.vue

```diff
@@ -128,10 +128,8 @@ export default {
     },
     convertedBalance() {
       if (this.isEthNetwork) {
-        const balance = BigNumber(this.balanceInETH).times(
-          this.ETHUSDValue.value
-        );
-        return `${this.ETHUSDValue.symbol + balance.toFixed(2).toString()}`;
+        const balance = BigNumber(this.balanceInETH).times(this.fiatValue);
+        return `${'$' + balance.toFixed(2).toString()}`;
```

Comment:
this can be a `getter` in `external` store as well

Reply:
done!